mkdocs.yml

```diff
@@ -495,6 +502,15 @@ nav:
               - Cluster Type: "user-guide/core-data-model/virtualization/clustertype.md"
               - Virtual Machine: "user-guide/core-data-model/virtualization/virtualmachine.md"
               - VM Interface: "user-guide/core-data-model/virtualization/vminterface.md"
+          - Wireless:
+              - "user-guide/core-data-model/wireless/wireless.md"
```

Comment:
Possibly `index.md`?

Reply:
I can rename, but this is the same as cloud.md